ribasim_qgis/core/nodes.py

```diff
@@ -125,19 +123,7 @@ def set_read_only(self) -> None:
 
     @property
     def renderer(self) -> QgsFeatureRenderer | None:
```

Comment:
Do we still need this function? Do we need to update the return type annotation to None only?